packages/nimble-components/src/combobox/index.ts

```diff
@@ -667,15 +632,12 @@ export class Combobox
      */
     protected override focusAndScrollOptionIntoView(): void {
         if (this.open) {
-            if (this.contains(document.activeElement)) {
-                this.control.focus();
-                if (this.firstSelectedOption) {
-                    requestAnimationFrame(() => {
-                        this.firstSelectedOption?.scrollIntoView({
-                            block: 'nearest'
-                        });
+            if (this.firstSelectedOption) {
```

Comment:
@jattasNI and @rajsitem the code I've removed here was partially responsible for keeping a test from passing. Essentially `document.activeElement` had not been updated to something contained within the `Combobox`. While I can't say I confidently know why that is, the removed logic still seems wholly unnecessary. I see no reason why we have to explicitly place focus on the `input` in the template when scrolling the selected option into view, nor a reason to gate this on the currently focused element already being with the `Combobox`. This method should just be concerned with scrolling the current selected item into view.